### PR TITLE
feat(run-lighthouse-tests): init [ED-325]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: actions/outputs.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as core from '@actions/core';
+import styles from 'ansi-styles';
+
+export function setOutput(name: string, value: unknown) {
+	core.info(
+		` -> ${styles.cyan.open}[${name}]${styles.cyan.close} ${JSON.stringify(value)}`,
+	);
```
Comment (StyleShit) [inline actions/outputs.ts:7]: 🔥

File: actions/run-lighthouse-tests/action.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Run Lighthouse tests
+description: Run Lighthouse tests on given URLs
+
+inputs:
+    urls:
+        description: A list of URLs to test, separated by new lines (e.g. `moshe:http://localhost:8889/moshe \n haim:http://localhost:8889/haim`)
```
Comment (StyleShit) [inline actions/run-lighthouse-tests/action.yml:6]:
```suggestion
        description: A map of URLs to test, with aliases as keys, separated by new lines (e.g. `moshe:http://localhost:8889/moshe \n haim:http://localhost:8889/haim`)
```

File: actions/run-lighthouse-tests/action.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        required: false
```
Comment (StyleShit) [inline actions/run-lighthouse-tests/action.yml:7]: why is it optional?

File: actions/run-lighthouse-tests/main.ts [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+const lhciConfig = {
+	ci: {
+		collect: {
+			headful: false,
+			numberOfRuns: 1,
+			url: [] as string[], // Will be filled in later.
+			settings: {
+				maxWaitForLoad: 90000,
+				throttlingMethod: 'devtools',
+				onlyCategories: [] as string[], // Will be filled in later.
```
Comment (StyleShit) [inline actions/run-lighthouse-tests/main.ts:33]:
```suggestion
				onlyCategories: [] as Array<typeof AVAILABLE_CATEGORIES[number]>, // Will be filled in later.
```

maybe?

File: actions/run-lighthouse-tests/main.ts [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+					'install',
+					'-g',
+					`@lhci/cli@${LHCI_VERSION}`,
+				]);
+			});
+		}
+
+		// eslint-disable-next-line @typescript-eslint/require-await
+		await core.group('Declare reports path', async () => {
+			setOutput(`reports-path`, OUTPUT_DIR);
```
Comment (StyleShit) [inline actions/run-lighthouse-tests/main.ts:59]:
```suggestion
			setOutput('reports-path', OUTPUT_DIR);
```

File: actions/run-lighthouse-tests/main.ts [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+					await fs.writeJSON('.lighthouserc.json', config);
+				},
+			);
+
+			await core.group(
+				`Running lighthouse-ci test on "${urlAlias}"`,
+				async () => {
+					await exec.exec('rm', ['-rf', OUTPUT_DIR]);
+					await exec.exec('rm', ['-rf', './.lighthouseci']);
```
Comment (StyleShit) [inline actions/run-lighthouse-tests/main.ts:104]:
```suggestion
					await exec.exec('rm', ['-rf', OUTPUT_DIR, './.lighthouseci']);
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: actions/run-lighthouse-tests/action.yml (hunk 2 above)
Comment (Nevoss) [reply actions/run-lighthouse-tests/action.yml:6]: Done

File: actions/run-lighthouse-tests/action.yml (hunk 3 above)
Comment (Nevoss) [reply actions/run-lighthouse-tests/action.yml:7]: Done

File: actions/run-lighthouse-tests/main.ts (hunk 4 above)
Comment (Nevoss) [reply actions/run-lighthouse-tests/main.ts:33]: Done

File: actions/run-lighthouse-tests/main.ts (hunk 5 above)
Comment (Nevoss) [reply actions/run-lighthouse-tests/main.ts:59]: Done

File: actions/run-lighthouse-tests/main.ts (hunk 6 above)
Comment (Nevoss) [reply actions/run-lighthouse-tests/main.ts]: Done

**Review by StyleShit — APPROVED** [review]

**Pull request merged**